manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrole.yaml

```diff
@@ -36,10 +36,12 @@ rules:
   resources:
   - workflows
   - workflows/finalizers
+  - workflowtemplates
+  - workflowtemplates/finalizers
```

Comment:
I think instead of adding to existing rule, it will be better to create a new rule entry so that the controller only has `get`, `list`, and `watch` on templates.